openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/postgresConnection.json

```diff
@@ -94,5 +100,5 @@
     }
   },
   "additionalProperties": false,
-  "required": ["hostPort", "username"]
+  "required": ["hostPort", "username", "database"]
```

Comment:
@ulixius9 will the `database` be a required field even in case when `ingestAllDatabases` is set to true?

Reply:
Yes it will be required, when this is similar to what we do for redshift even if we want to ingest all databases we will have to pass at least one database name